extension/src/plots/model/index.ts

```diff
@@ -171,6 +182,58 @@ export class PlotsModel extends ModelWithPersistence {
     }
   }
 
+  public getCustomPlots() {
+    if (!this.customPlots) {
+      return
+    }
+    return {
+      plots: this.customPlots,
+      size: this.getPlotSize(Section.CUSTOM_PLOTS)
+    }
+  }
+
+  public recreateCustomPlots() {
+    const customPlots: CustomPlotData[] = collectCustomPlotsData(
+      this.getCustomPlotsOrder(),
+      this.experiments.getExperiments()
+    )
+    this.customPlots = customPlots
```

Comment:
[B] This needs to update when experiments are selected/de-selected:


https://user-images.githubusercontent.com/37993418/221741109-ba94ff41-5cf5-4088-8df9-23bfd12cadee.mov

Reply:
The custom plots are supposed to hold all collected experiments, not just selected ones. If I remember correctly from the discussion, we wanted the custom plots to hold all experiments so that they could hold more than 7 🤔. 

I can try to make this more clear in the info tooltip, or we can discuss further if you think it's better for the plots to behave like the other plots in the webview?

Reply:
Though I just realized, we'd need the custom plots to update if a user deletes an experiment if we choose to keep this logic.